hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaControllerManagement.java

```diff
@@ -693,11 +693,16 @@ private static void copy(final Map<String, String> src, final Map<String, String
         if (src == null || src.isEmpty()) {
             return;
         }
-        src.entrySet().forEach(e -> {
-            if (e.getValue() != null) {
-                trg.put(e.getKey(), e.getValue());
+
+        src.forEach((key, value) -> {
+            if (key != null) {
```

Comment:
I think this check (and the corresponding `else`) is not needed. This should be covered by `@NotNull`, right?